tests/Resources/date.test.js

```diff
@@ -95,8 +95,8 @@
 		should(date.toLocaleTimeString([ 'en-US' ], options)).be.a.String();
 		should(date.toLocaleTimeString([ 'en-US', 'de-DE' ], options)).be.a.String();
 
-		should(date.toLocaleTimeString('en-US', options)).be.eql('8:02:05 PM');
-		should(date.toLocaleTimeString('de-DE', options)).be.equalOneOf([ '8:02:05 PM', '8:02:05 nachm.' ]);
+		should(date.toLocaleTimeString('en-US', options)).be.equalOneOf(['8:02:05 PM', '8:02:05 in the evening']);
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/Resources/d..., tests/Resources/d...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/date.test.js;
  Line: 98;
-->

- :warning: tests/Resources/date.test.js line 98 – A space is required after '['. (array-bracket-spacing)
- :warning: tests/Resources/date.test.js line 98 – A space is required before ']'. (array-bracket-spacing)